### PR TITLE
feat(container): add settings container

### PR DESCRIPTION
**Pull request by artyorsh**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

 #### Short description of what this resolves:

* Integrates settings container
* Integrates Edit Profile (x3) containers
* Refactors icons to separate dir

Here is one of demo ;)
drawing

### CODE REVIEW

File: src/containers/settings/editProfile/editProfile1/editProfile1.component.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      <ScrollView style={themedStyle.container}>
+        <View style={themedStyle.photoSection}>
+          <ProfilePhoto
+            style={themedStyle.photo}
+            source={{ uri: profile.photo }}
+            button={this.renderPhotoButton}
+          />
+        </View>
+        <View style={themedStyle.infoSection}>
+          <ProfileParameter
```
Comment (32penkin) [inline src/containers/settings/editProfile/editProfile1/editProfile1.component.tsx:63]: May be loop by Object.keys()?

File: src/containers/settings/editProfile/editProfile2/editProfile2.component.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            style={themedStyle.photo}
+            shape='rounded'
+            source={{ uri: profile.photo }}
+            button={this.renderPhotoButton}
+          />
+        </View>
+        <View style={themedStyle.descriptionSection}>
+          <Text style={themedStyle.description}>{profile.about}</Text>
+        </View>
+        <View style={themedStyle.infoSection}>
```
Comment (32penkin) [inline src/containers/settings/editProfile/editProfile2/editProfile2.component.tsx:69]: Is it makes sense to make this like separate component?

File: src/containers/settings/editProfile/editProfile3/editProfile3.component.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+            <ProfileParameter
+              style={themedStyle.nameParameter}
+              value={profile.lastName}
+            />
+          </View>
+        </View>
+        <View style={themedStyle.descriptionSection}>
+          <Text style={themedStyle.description}>{profile.about}</Text>
+        </View>
+        <View style={themedStyle.infoSection}>
```
Comment (32penkin) [inline src/containers/settings/editProfile/editProfile3/editProfile3.component.tsx:78]: Third place of using the same code.

File: src/containers/settings/settings.component.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+const Section = (props?: SectionProps): React.ReactElement<TouchableOpacityProps> => {
+  return (
+    <TouchableOpacity
+      {...props}
+      activeOpacity={1.0}
+    />
+  );
+};
+
+const Divider = (props?: ViewProps): React.ReactElement<ViewProps> => {
```
Comment (32penkin) [inline src/containers/settings/settings.component.tsx]: I believe we have to create such component in the RNUK. We've missed it in the Menu and List components

File: src/core/navigation/routes.tsx [hunk 5]
```diff
@@ -39,10 +44,34 @@ const HeadingNavigationOptions = ({ navigation }): NavigationScreenConfig<Naviga
   return { ...navigation, header };
 };
 
+const EditProfileNavigator: NavigationContainer = createStackNavigator(
+  {
+    ['Edit Profile']: EditProfileContainer,
```
Comment (32penkin) [inline src/core/navigation/routes.tsx:49]: Looks like great solution.

**Review by 32penkin — CHANGES_REQUESTED** [review]

**Review by 32penkin — APPROVED** [review]

File: src/containers/settings/settings.component.tsx (hunk 4 above)
Comment (artyorsh) [reply src/containers/settings/settings.component.tsx]: It also makes sense to achieve this using borders, like in EditProfile components

**Pull request merged**